src/app/shared/components/template/components/toggle-bar/toggle-bar.scss

```diff
@@ -9,34 +9,13 @@ ion-toggle {
 
 ion-toggle {
   --track-background: #b4b7b7;
```

Comment:
This colour should be read from our theming system rather than being hardcoded. I haven't shown you this yet, but try replacing this hex value with `var(--ion-color-gray-200)`, and then experiment with different values instead of `200` to find a grey colour that looks right. Using a variable in this way means that when the global theme is changed, this colour will update automatically

Reply:
Done so, there seemed to be no need to experiment.